### DISCUSSION TITLE
Any breaking changes to be expected in the near future?

### DISCUSSION THREAD

**Discussion by Jonas1893** [hmlongco/Factory#141] [Q&A] [answered]
Hi,

we are working on a big project where `Resolver` is currently used in multiple pods. We want to migrate to `Factory` but since the migration effort will be high: are there any breaking changes expected to `Factory` (potentially caused by Swift / platform updates in the Xcode betas) in the near future or is it now a good time to do the migration? 

Thank you

**Comment by hmlongco** [comment, marked as answer]
There are no breaking changes planned, nor are there any known issues in regard to Xcode 15.

I'd say go for it. ;)

**Reply by Jonas1893** [reply]
Thank you for the quick response!
